content/en/about/integration.md

```diff
@@ -0,0 +1,57 @@
+---
+type: docs
+category: About Sigstore
+description: Integration
+home: true
+title: Integration
+weight: 3
+---
+
+![Sigstore](sigstore-logo_horizontal-color.svg)
+
+## Integration
+Many of the recent high-profile software attacks that have alarmed open-source users globally were consequences of supply chain integrity vulnerabilities: attackers gained control of a build server to use malicious source files, inject malicious artifacts into a compromised build platform, and bypass trusted builders to upload malicious artifacts. Sigstore provides a way to sign, verify, and log artifacts to ensure that they are from who they say they are and to shore up supply chain vulnerabilities.  
+
```

Comment:
I'd like a paragraph about package repository adoption too. Maybe we could reuse some of the language from the roadmap around this? https://github.com/sigstore/community/blob/main/ROADMAP.md#adoption-patterns-for-sigstore

Reply:
i put something in but am not sure it's the right info.  please check.

Reply:
@haydentherapper did i get the right information?

Reply:
Thanks for updating. I'll merge for now but if anything we can change later